README.md

```diff
@@ -277,6 +277,136 @@ q4 = StructSimilarityQuery(structure_search_type="file_upload",
+
+Examples of how to instantiate Residues can be found below. These can then be put into a list and passed through to a Structure Motif Query.
+```python
+from rcsbsearchapi.search import StructureMotifResidue
+
+# construct a Residue with a Chain ID of A, an operator of 1, a residue 
+# number of 192, and Exchanges of "LYS" and "HIS"
+Res1 = StructureMotifResidue("A", "1", 192, ["LYS", "HIS"])
+# as for what is a valid "Exchange", the package provides these as a literal,
+# and they should be type checked. 
+
+# you can also use positional arguments:
+# this query is the same as above. 
+Res2 = StructureMotifResidue(struct_oper_id="1", chain_id="A", exchanges=["LYS", "HIS"], label_seq_id=192)
+
+# after delcaring a minimum of 2 and as many as 10, they can be passed into a list for use in the query itself:
+Res3 = StructureMotifResidue("A", "1", 162)  # exchanges are optional
+
+ResList = [Res1, Res3]
+```
+From there, these Residues can be used in a query. As stated before, you can only include 2 - 10 residues in a query. If you fail to provide residues for a query, or provide the wrong amount, the package will throw a ValueError. 
+
+For a Structure Motif Query using an entry_id, the only other necessary value that must be passed into the query is the residue list. The default type of query is an entry_id query. 
+
+As this type of query has a lot of optional parameters, do *not* use positional arguments as more than likely an error will occur. 
+
+Below is an example of a basic entry_id Structure Motif Query, with the residues declared earlier:
+```python
+from rcsbsearchapi.search import StructMotifQuery
+
+q1 = StructMotifQuery(entry_id="2MNR", residue_ids=ResList)
+list(q1())
+```
+Like with Structure Similarity Queries, a file url or filepath can also be provided to the program. These can take the place of an entry_id. 
+
+For a file url query, you *must* provide both a valid file URL (a string), and the files file extension (also as a string). Failure to provide these elements correctly will cause the package to throw an AssertionError. 
+
+Below is an example of the same query as above, only this time providing a file url:
+```python
+link = "https://files.rcsb.org/view/2MNR.cif"
+q2 = StructMotifQuery(querytype="file_url", url=link, file_extension="cif", residue_ids=ResList)
+# querytype MUST be provided. A mismatched query type will cause an error. 
+list(q2())
+```
+Like with Structure Similarity Queries, a filepath to a file may also be provided. This file must be a valid file accepted by the search API. A file extension must also be provided with the file upload. 
+
+The query would look something like this. Note that this is abstracted for the purpose of notebook portability.
+```python
+filepath = "/absolute/path/to/file.cif"
+q3 = StructMotifQuery(querytype="file_upload", filepath=filepath, file_extension="cif", residue_ids=ResList)
```

Comment:
Sorry I didn't catch this earlier, but can you use the same keyword names as used in `StructSimilarityQuery` where possible (except the `asym_id` thing we talked about)? So, `querytype` would be `structure_search_type` and `filepath` would be `file_path`.